tests/data/placebo/test_ecs_service_update/ecs.DescribeClusters_1.json

```diff
@@ -0,0 +1,126 @@
+{
+    "status_code": 200,
+    "data": {
+        "clusters": [
+            {
+                "clusterArn": "arn:aws:ecs:us-east-1:644160558196:cluster/test-cluster",
```

Comment:
there's *alot* of extraneous data in these recordings, either extra remove by hand or re-record in an empty region to reduce.